pkg/backup/client.go

```diff
@@ -122,6 +122,11 @@ func (bc *Client) GetTS(ctx context.Context, duration time.Duration, ts uint64)
 	return backupTS, nil
 }
 
+// SetLockFile set write lock file.
+func (bc *Client) SetLockFile(ctx context.Context) error {
+	return bc.storage.Write(ctx, utils.LockFile, []byte{})
```

Comment:
Could you write some warnings to the file? Eg. DO NOT DELETE and explain what the file does.